api/bazel/repository_locations.bzl

```diff
@@ -14,11 +14,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "protoc-gen-validate (PGV)",
         project_desc = "protoc plugin to generate polyglot message validators",
         project_url = "https://github.com/envoyproxy/protoc-gen-validate",
-        version = "872b28c457822ed9c2a5405da3c33f386ac0e86f",
-        sha256 = "388ea2261bc1d2c6ef6ec01bfaa3aec451aedb245e23514033ccc9b5cc10c4ab",
+        version = "9db8e779b46119eef7d67e3d1120c118d593611c",
+        sha256 = "a3bfcf965499bfb17299a6931325eab653f99306dac413a5205e4b65e6ab027d",
         strip_prefix = "protoc-gen-validate-{version}",
         urls = ["https://github.com/envoyproxy/protoc-gen-validate/archive/{version}.tar.gz"],
-        release_date = "2021-01-05",
+        release_date = "2021-03-16",
```

Comment:
We've alternated between releases and commits with PGV but @akonradi cut `v0.5.0` of PGV yesterday - can we switch to that? https://github.com/envoyproxy/protoc-gen-validate/releases/tag/v0.5.0. From a dependency policy point of view we prefer releases.
```python
    com_envoyproxy_protoc_gen_validate = dict(
        project_name = "protoc-gen-validate (PGV)",
        project_desc = "protoc plugin to generate polyglot message validators",
        project_url = "https://github.com/envoyproxy/protoc-gen-validate",
        version = "0.5.0",
        sha256 = "b3f51da19196174fca07d947adcadf478d6825ea35d3261c9562532f38f0ea94",
        release_date = "2021-03-15",
        strip_prefix = "protoc-gen-validate-{version}",
        urls = ["https://github.com/envoyproxy/protoc-gen-validate/archive/v{version}.tar.gz"],
        use_category = ["api"],
        implied_untracked_deps = [
            "com_github_iancoleman_strcase",
            "com_github_lyft_protoc_gen_star",
            "com_github_spf13_afero",
            "org_golang_google_genproto",
            "org_golang_x_text",
        ],
    ),
```

Reply:
Unfortunately, this release [doesn't include the fix to some of the presubmit failures](https://github.com/envoyproxy/protoc-gen-validate/compare/v0.5.0...main), specifically [this commit](https://github.com/envoyproxy/protoc-gen-validate/commit/9db8e779b46119eef7d67e3d1120c118d593611c).